.eslintrc

```diff
@@ -0,0 +1,7 @@
+extends:
+  - eslint:recommended
+  - airbnb
+
+rules:
+  no-console: 1
+  no-var: 0
```

Comment:
Silencing all the errors about use of `var` usage from the AirBnB style ruels, but I can convert all the `var`s to `let`s and `const`s if we want.